2026/README.md

```diff
@@ -0,0 +1,16 @@
+# Interop 2026
+
+**Interop 2026 is now open for proposals!** See the [proposal guide](../making-a-proposal.md) for details, and how to give your proposal the best chance of success.
```

Comment:
This file is from https://github.com/web-platform-tests/interop/pull/989 aside from a tweak to this line